app/packs/controllers/datatable_works_controller.js

```diff
@@ -12,9 +12,12 @@ export default class extends Controller {
       { select: [1, 5, 6, 7, 8], sortable: false } // Disable unsortable columns
     ]
     if (this.hideDepositorValue) columns.push({ select: 2, hidden: true})
-    new DataTable("#worksTable", {
+    const dt = new DataTable("#worksTable", {
       columns: columns,
       searchable: true
     })
+    dt.on('datatable.page', () => {
```

Comment:
Can you drop a comment here about what this does?

Reply:
Yup. Done.